components/shared/indexIssueDetails/IndexIssueDetails.js

```diff
@@ -166,7 +166,7 @@ export default function IndexIssueDetails({
   return (
     <>
       <BackLink indexType={indexType} history={router} />
```

Comment:
This is solidly outside the scope of your issue, but you did mention the back button, so FWIW:

This took me a second to figure out, because this link actually takes you back like the back button, not a forward navigation to the index of the current resources being viewed. That's not what I normally think of a link-on-a-page doing, but 🤷🏼 it's going to do the right thing if you browse down to a product or project.

Put another way - If you enter into the site with a deep link (like in your PR comment), the back button takes you back to github or whatever external site the browser was on beforehand. 

On 170, there is logic handling the difference between products and projects -- what do you think about making a nextjs `<Link>` to `/products` or `/projects` which would replace the bespoke `<BackLink>` component. I think this would make it feel more normal, but it's also, super not a big deal.

Reply:
Good call, Frank! I agree that this is an existing bug, and I also would expect that link to always go to the Products page (and not assume that the Products or Projects page was the last page visited). It could be a bug issue, but, could be nice to patch up here if it doesn't get too out of control.

Reply:
I third the motion

Reply:
Great catch Frank! I didn't realize this is how the button functions, I think this should definitely be addressed and I think I could just address it in this issue. This is why I love PR reviews